src/app/shared/helpers/app-helpers.ts

```diff
@@ -2,4 +2,10 @@ export function getRandomInt(minInclusive: number, maxExclusive: number) {
   minInclusive = Math.ceil(minInclusive);
   maxExclusive = Math.floor(maxExclusive);
   return Math.floor(Math.random() * (maxExclusive - minInclusive + 1)) + minInclusive;
+}
+
+export function removeParenthesis(s: string) {
```

Comment:
created a helper function to remove parenthesis as window.open will remove parenthesis in the url and then the component cannot find the value without parenthesis.